spec/features/maintaining_a_visit_spec.rb

```diff
@@ -35,10 +39,10 @@
     cassette_name: :maintianing_a_visit_viewing_rejected
   } do
     visit visit_path(id: 'FOOBAR', locale: 'en')
-    expect(page).to have_text('Your visit request cannot take place')
+    expect(page).to have_text('Your visit request has been rejected')
 
-    click_link 'new visit'
-    expect(page).to have_css('h1', text: 'Who are you visiting?')
+    # click_link 'new visit'
```

Comment:
@KPobeeNorris - should these lines be commented out

Reply:
@TobyRet - As far as I can see this was commented out as part of @chrishanes's work yesterday. @chrishanes I tested it again with these lines back in and the test passes; are you ok with them being added in?

Reply:
@TobyRet @KPobeeNorris Sorry, yes those were supposed to be uncommented.